Library/Homebrew/cmd/test-bot.rb

```diff
@@ -809,61 +805,66 @@ def test_ci_upload(tap)
 
     if pr
       if ARGV.include?("--legacy")
-        pull_pr = "https://github.com/Homebrew/legacy-homebrew/pull/#{pr}"
+        pull_pr = "https://github.com/Homebrew/homebrew/pull/#{pr}"
```

Comment:
I think this change was made accidentally and is unintended. (Not that it matters much as `--legacy` is used very infrequently by now.)